components/tools/OmeroWeb/test/integration/test_api_containers.py

```diff
@@ -156,6 +171,22 @@ def screen_plates(self, user1):
 
         screen = get_update_service(user1).saveAndReturnObject(screen)
         plate = get_update_service(user1).saveAndReturnObject(plate)
+
+        # Add well to first plate
```

Comment:
i will add a plate acquisition to the test data since it is what will happen by default with a plate import